tests/models/test_base.py

```diff
@@ -41,29 +47,28 @@ def test_fiware_header(self):
         self.assertRaises(ValidationError, FiwareHeader,
                           service='filip', service_path='#')
         headers = FiwareHeader.parse_obj(self.fiware_header)
-        with ContextBrokerClient(fiware_header=headers) as client:
+        with ContextBrokerClient(url=settings.CB_URL,
+                                 fiware_header=headers) as client:
             entity = ContextEntity(id='myId', type='MyType')
             for path in self.service_paths:
                 client.fiware_service_path = path
                 client.post_entity(entity=entity)
                 client.get_entity(entity_id=entity.id)
             client.fiware_service_path = '/#'
-            self.assertEqual(len(client.get_entity_list()),
-                             len(self.service_paths))
+            self.assertGreaterEqual(len(client.get_entity_list()),
+                                    len(self.service_paths))
             for path in self.service_paths:
                 client.fiware_service_path = path
                 client.delete_entity(entity_id=entity.id)
 
     def tearDown(self) -> None:
         # Cleanup test server
-        with ContextBrokerClient() as client:
-            client.fiware_service = 'filip'
+        with ContextBrokerClient(url=settings.CB_URL) as client:
+            client.fiware_service = settings.FIWARE_SERVICE
 
             for path in self.service_paths:
-                try:
-                    client.fiware_service_path = path
-                    entities = client.get_entity_list()
-                    client.update(entities=entities, action_type='delete')
-                except RequestException:
-                    pass
+                header = FiwareHeader(
+                    service=settings.FIWARE_SERVICE,
+                    service_path=path)
+                clear_all(fiware_header=header, cb_url=settings.CB_URL)
```

Comment:
Ok here is a good use for the teardown methode, as it cleans the generated paths.